lesson9/index.js

```diff
@@ -0,0 +1,40 @@
+const ul = document.getElementById('js-lists');
+
+const listData = [
+    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+]
+
+async function promise() {
+    return new Promise(resolve => {
+        const lodingImage = document.createElement('img');
+        lodingImage.src = './loading-circle.gif';
+        ul.appendChild(lodingImage);
```

Comment:
create function and run them.

Reply:
Thank you for your review.
I created new function `displayLoadingImage`.
commit: c75f2c9